client/elements/suttaplex/sc-parallel-item.html

```diff
@@ -148,14 +175,31 @@
                           'skt': 'Sanskrit',
                           'bo': 'Tibetan'
                       }
+                  },
+                  // Responsive layout variables that contain css classes.
+                  // These are changed when iron-media-query detects a layout width change.
+                  mainInfoContainerClasses: {
+                      type: String,
+                      value: ''
                   }
               }
           }
 
+          ready() {
+              super.ready();
+              window.addEventListener('wide-layout-changed', this._screenWidthChanged);
+          }
+
+          _screenWidthChanged() {
+              console.log('widht changed');
+              if (this.wideLayout) {
+                  this.mainInfoContainerClasses = 'right-margin-md';
+              } else this.mainInfoContainerClasses = '';
+          }
+
           // returns the relevant header, either the title in the translated language,
           // otherwise in pali or otherwise just the ID.
           _computeTitle() {
-              // TODO internationalization: return title in preferred language
               if (this.parallelItem.original_title) {
                   return this.parallelItem.original_title;
               }
```

Comment:
Maybe we can use conditional operator here 
`return this.parallelItem.original_title ? this.parallelItem.original_title : this.parallelItem.uid`